### PR TITLE
Independent Assistants

### PR DESCRIPTION
**Pull request by Ben10083**
Assistants are now possible as independents. Taken from [this](https://forums.aurorastation.org/topic/19574-bring-back-independent-assistants/) forum post

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Comment by Ben10083** [comment]
This says so much about society

**Comment by Arrow768** [comment]
The "maintainer discussion" and "do not merge" labels were not considered when this PR has been queued for merge.

That maintainer discussion has since happened, and we decided not to revert it for the following reasons:

**Regulation issues**
As currently defined in our regs assistants are subordinate to *any* crew member. (see the chain of command on the wiki)
Accordingly, "independent" assistants would also be beholden to that chain of command, therefore rendering them rather useless as "consular assistants".

This *could* potentially be fixed by updating the regulations for "independent" assistants.
However that leads us to the second issue.

**Assistant Capacity**
If we added a exception to our regulations that independent assistants are not beholden to the chain of commands this leaves us with the issue of "assistant capacity".
As implemented in this PR, there are no mechanical restrictions that would prevent a chaplain from bringing their personal choir. Or (any other position) from having a multitude of assistants.

And finally, there is also the issue of 

**Leading Job Presence**
With the current implementation, there are no mechanical restrictions that prevent a independent assistant from spawning if the leading role (i.e. consular) isnt present, therefore creating a unwhitelisted entity that's almost entirely outside the chain of command (if the above mentioned regulation changes are applied).


As such the PR has been reverted and any future implementation of "consular assistants" would need to have the following features:
* Distinct Role/Job
* Limited to 1 Assistant per consular
* Jobslot manually enabled by the consular.
* The same species/origin/... restrictions applied as to the consular (with the exception of vaurca where the assistant can be a worker, but can not be a breeder)

**Pull request merged**
